### PR TITLE
Conformance results for OpenShift 4.0 (Kube 1.12)

### PR DESCRIPTION
**Pull request by smarterclayton**

### CODE REVIEW

File: v1.12/openshift/PRODUCT.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+vendor: Red Hat
+name: OpenShift Online and OpenShift Container Platform by Red Hat
+version: v4.0.0
+website_url: https://www.openshift.com
+documentation_url: https://docs.openshift.com
+product_logo_url: https://assets.openshift.net/content/subdomain/openshift-logo-horizontal.svg
```
Comment (dankohn) [inline v1.12/openshift/PRODUCT.yaml]: From https://www.openshift.com/about/logos it looks like you've switched OpenShift logos to include the Red Hat name.

The one we have at https://landscape.cncf.io/logos/red-hat-open-shift.svg looks correct.

Comment (smarterclayton) [reply v1.12/openshift/PRODUCT.yaml]: Updated to 
https://cdn2.hubspot.net/hubfs/4305976/images/openshift-legacy/logos/openshift/Logotype_RH_OpenShift_wLogo_RGB_Black.svg which is the current brand guideline.

**Comment by taylorwaggoner** [comment]
@smarterclayton, the results look great, but could you please remove the version.txt file?  It is no longer required per these guidelines:https://github.com/cncf/k8s-conformance/blob/master/instructions.md#contents-of-the-pr.  Thanks!

**Comment by smarterclayton** [comment]
Updated, PTAL.

**Comment by taylorwaggoner** [comment]
You are now Certified Kubernetes

**Pull request merged**